app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/UpscanConnector.scala

```diff
@@ -123,55 +104,6 @@ class UpscanConnectorImpl @Inject() (
   private val minFileSize: Int  = getUpscanInitiateConfig[Int]("min-file-size")
```

Comment:
This can be removed as we don't use it

Reply:
Needs to be removed from application.conf + the app-config-* files

Reply:
is it not needed in the `initiate` method below?

Reply:
ah, I see it defaults to 0 anyway,  I'll get rid of it

Reply:
done